src/ShowGuesser.test.tsx

```diff
@@ -107,4 +112,34 @@ describe('<ShowGuesser />', () => {
       expect(screen.queryAllByText('Body')).toHaveLength(1);
     });
   });
+
+  test('renders with tabs', async () => {
+    render(
+      <AdminContext dataProvider={dataProvider}>
+        <SchemaAnalyzerContext.Provider value={hydraSchemaAnalyzer}>
+          <ShowGuesser resource="users" id="/users/123">
+            <Tab label="Tab 1">
+              <TextField source="id" label="label of id" />
+              <TextField source="title" label="label of title" />
+            </Tab>
+            <Tab label="Tab 2">
+              <TextField source="body" label="label of body" />
+            </Tab>
+          </ShowGuesser>
+        </SchemaAnalyzerContext.Provider>
+      </AdminContext>,
+    );
+
+    await waitFor(() => {
+      expect(screen.queryAllByRole('tab')).toHaveLength(2);
+      // First tab.
+      expect(screen.queryAllByText('label of id')).toHaveLength(1);
+      expect(screen.queryAllByText('/users/123')).toHaveLength(1);
+      expect(screen.queryAllByText('label of title')).toHaveLength(1);
+      expect(screen.queryAllByText('Title')).toHaveLength(1);
+      // Second tab, unavailable.
```

Comment:
It would be nice to then click on the tab and verify that the body is now accessible.

Reply:
OK, I will add it.